test/spec/spec.controller.js

```diff
@@ -640,6 +640,108 @@ describe('lib/base-controller', () => {
       });
     });
 
+    describe('getErrors', () => {
+
+      let req;
+      let res;
+      let errors;
+
+      beforeEach(() => {
+
+        errors = {
+          key1: {
+            key: 'key1',
+            type: 'required'
+          },
+          key2: {
+            key: 'key2',
+            type: 'maxlength',
+            arguments: [10]
+          },
+          key3: {
+            key: 'key3',
+            type: 'before',
+            arguments: [3, 'years']
+          }
+        };
+
+        req = request();
+        req.form = {};
+        res = response();
+        sinon.stub(controller, 'getErrors').returns(errors);
+        sinon.stub(controller, 'getErrorMessage', e => `error for ${e.key}`);
```

Comment:
What's your opinion on standardising how we represent error arguments, amongst others? 
I notice that there is often more than one style used throughout all repos.
E.g. For errors I tend to use `err`, and you `e`.